### ISSUE TITLE
Doesn't work with ActiveRecord 6

### ISSUE THREAD

**Issue by joevandyk** [Envek/after_commit_everywhere#5]
https://github.com/Envek/after_commit_everywhere/commit/0c393f238baa2d0d1f507e1143ab0a08aa0ead9a adds the code to test AR 6

all the tests fail with errors like

```
  1) AfterCommitEverywhere#after_commit within transaction executes code only after commit
     Failure/Error:
       ActiveRecord::Base.transaction do
         subject
         expect(handler).not_to have_received(:call)
       end

     NoMethodError:
       undefined method `trigger_transactional_callbacks?' for #<AfterCommitEverywhere::Wrap:0x00007f97cb078748>
     # ./spec/after_commit_everywhere_spec.rb:27:in `block (4 levels) in <top (required)>'
```

**Comment by joevandyk** [comment]
https://github.com/Envek/after_commit_everywhere/compare/master...joevandyk:rails6 makes all but one test work:

```
Failures:

  1) use_transactional_tests=true raises with transaction without after_commit
     Failure/Error:
       expect { ActiveRecord::Base.transaction { subject } }.to \
         raise_error(Isolator::UnsafeOperationError)

       expected Isolator::UnsafeOperationError but nothing was raised
     # ./spec/transactional_tests_spec.rb:44:in `block (2 levels) in <top (required)>'

Finished in 0.028 seconds (files took 1.23 seconds to load)
31 examples, 1 failure
```

**Comment by Envek** [comment]
Hello! Thank you very much for investigation! I hope I will be able to look and fix it in a day or two. 

P.S> And why not to create a Pull Request with your work?

**Comment by joevandyk** [comment]
https://github.com/Envek/after_commit_everywhere/pull/6 - there's still one failing test, something to do with Isolate.

**Comment by Envek** [comment]
Fixed in 0.1.4
